mora/service/address.py

```diff
@@ -147,9 +149,110 @@ def convert(start, end, effect):
                 ),
                 key=(
                     lambda v: (
-                        common.get_valid_from(v),
-                        locale.strxfrm(v[keys.ADDRESS_HREF]),
+                        common.get_valid_from(v) or util.negative_infinity,
+                        common.get_valid_to(v) or util.positive_infinity,
+                        str(v[keys.ADDRESS_PRETTY]),
                     )
                 ),
             ),
         )
+
+
+def get_relation_for(classobj, value, start, end):
+    scope = classobj['scope']
+
+    effect = {
+        'from': util.to_lora_time(start),
+        'to': util.to_lora_time(end),
+    }
+
+    if scope == 'DAR':
+        if not util.is_uuid(value):
+            raise ValueError('{!r} is not a valid address UUID!'.format(value))
+
+        return {
+            'uuid': value,
+            'objekttype': classobj['uuid'],
+            'virkning': effect,
+        }
+
+    elif scope in addr.URN_FORMATS:
+        return {
+            'urn': addr.URN_FORMATS[scope].format(value),
+            'objekttype': classobj['uuid'],
+            'virkning': effect,
+        }
+
+    else:
+        raise ValueError('unknown address scope {!r}!'.format(scope))
+
+
+def create_address(employee_uuid, req):
```

Comment:
We should probably decide on whether to do it like this, or like creating IT systems, and generalize it, considering how similar the procedures are.

Reply:
Agreed, I intend to do so in a followup change.